scripts/monorepo/runPublished.js

```diff
@@ -23,23 +23,26 @@ const websitePackages = [
   '@fluentui/api-docs',
 ];
 
+const releaseScope = process.env.RELEASE_VNEXT ? 'v9' : 'v8';
```

Comment:
can we add more docs to this  ? there is something for beachbalPackagesScopes but Im not sure its up to date